packages/notifier/test/iterable-testing-tools.js

```diff
@@ -297,8 +297,7 @@ export const carol = async subscriptionP => {
     harden({
       updateState: val => {
         if (val === 'a') {
-          // @ts-expect-error
```

Comment:
always happy to see errors go away. I'll assume it's not because some part is `any` now.

Reply:
I thought I didn't introduce an `any`, but I inadvertantly did.  I'll fix this.